vignettes/Intro_to_jmpost.Rmd

```diff
@@ -0,0 +1,419 @@
+---
+title: "Introduction to `jmpost`"
+subtitle: "`J`oint `M`odel for `P`osterior `O`verall `S`urvival `T`ranjectories"
+package: jmpost
+author:
+  - Georgios Kazantzidis
+  - Daniel Sabanés Bové
+  - Xiaoyan Fang
+header-includes: 
+  - \usepackage{tikz}
+  - \usepackage{pgfplots}
+  - \usetikzlibrary{mindmap}
+  - \usetikzlibrary{positioning}  
+  - \usetikzlibrary{shapes.geometric, arrows}   
+output:
+  pdf_document: default
+bibliography: bibliography.bib
+vignette: |
+  %\VignetteIndexEntry{Introduction to `jmpost`}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+ 
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = F
+)
+library(jmpost)
+```
+# Introduction
+
+In clinical development of new cancer medicines a key challenge is to decide on the next steps after having collected the initial data in Phase 1 studies. 
+In phase 1 studies patients are followed until the cancer is growing again (progression event). 
+Thus, reliable overall survival data are not available. 
+However, we can use additional information from previous clinical trials or real-world data to understand the time between progression and death.
+Specifically, we can correlate the tumor response data, that are longitudinal measurements, (i.e. the shrinkage and growth of tumor lesions over time) with the overall survival of the patients or their hazard ratios [@beyer2020multistate]. 
+Thereby we can predict the overall survival in our Phase 1 studies and therefore make better decisions [@kerioui2020bayesian].
+
+The sum of the longest diameter of the cancer (SLD) provides an effective and representative biomarker for longitudinal measurements in solid cancer. 
+SLD is relatively easy to measure with great accuracy while also being a good representative of the development of cancer. 
+
+The association between the longitudinal data and the overall survival can be achieved in different ways.
+The predicted SLD value or its first derivative over time can be directly treated as the linking factor.
+Other factors such as the time to growth, the tumor shrinkage rate or the tumor growth rate can also be used to associate longitudinal outcomes with overall survival.
+
+Bayesian methodology provides an important tool for the analysis of similar data.
+Firstly, the use of historical data from previous trials is improving the performance of the models.
+Secondly, prior definition allows the adjustment of the models according to the specific factors of each analysis.
+
+Although these statistical methods provide significant advantages in decision making processes in cancer drug development, their implementation is relatively intricate and time consuming.
```

Comment:
cite 2-3 other R-packages, like `jmbayes` or there is an INLA package for joint modeling